cassandra-3.x-scenario/config/docker-compose.yml

```diff
@@ -0,0 +1,39 @@
+version: '2.1'
+services:
+  cassandra-server:
+    image: cassandra:3.11.4
+    expose:
+      - "9042"
+
+  cassandra-scenario:
+    image:  skywalking/cassandra-scenario:{CASES_IMAGE_VERSION}
+    expose:
+      - "8080"
+    ports:
+      - "{SERVER_OUTPUT_PORT}:8080"
+    environment:
+      - COLLECTOR_SERVER=mock-collector:19876
+      - CASSANDRA_HOST=cassandra-server
+      - CASSANDRA_PORT=9042
+    links:
+      - mock-collector
+      - cassandra-server
+    volumes:
+     - {AGENT_FILE_PATH}:/usr/local/skywalking-cassandra-scenario/agent
+    depends_on:
+      mock-collector:
+        condition: service_healthy
+
+  mock-collector:
+    image: skyapm/mock-collector:{COLLECTOR_IMAGE_VERSION}
+    environment:
+      - GRPC_BIND_HOST=mock-collector
+    expose:
+      - "12800"
+    ports:
+      - "{COLLECTOR_OUTPUT_PORT}:12800"
+    healthcheck:
+      test: ["CMD", "curl", "-f", "http://127.0.0.1:12800/receiveData"]
```

Comment:
Have you used this to check server status?

Reply:
no, how to use it to check server status ? is there a sample to follow ?